src/wallet/rpcwallet.cpp

```diff
@@ -775,11 +775,14 @@ static UniValue getbalance(const JSONRPCRequest& request)
     }
 
     isminefilter filter = ISMINE_SPENDABLE;
+    bool include_watchonly = false;
```

Comment:
15747: should drop `filter` like in the original PR

Reply:
Must have been confused, fixed